news/2015-11-25-arq-container-se-CR1.asciidoc

```diff
@@ -0,0 +1,19 @@
+---
+layout: news
+title: Arquillian container SE 1.0.0.CR1 released!
+author: Tomas Remes
+priority: 1
+change_frequency: weekly
+---
+
+We are pleased to announce new release of Arquillian container SE. If you are looking for reliable way to test your application in Java SE environment
+then this is the right tool to try out!
+
+This managed SE container launches new JVM process for each of your test archives. Easily configurable classpath
+comes very handy and all you need to do is to specify path to your directory containing your dependencies jars via container property. You can get ready your
```

Comment:
I would add two things:
1. it's possible to set a system property (very useful and frequently used in SE imho)
2. you don't need the directory with dependencies if ShrinkWrap API is enough